modules/cactus-web/src/ConfirmModal/ConfirmModal.tsx

```diff
@@ -100,6 +100,7 @@ export const ConfirmModal = styled(ConfirmModalBase)`
     }
   }
   .children {
+    overflow-wrap: break-word;
```

Comment:
Despite assurances from MDN, this still doesn't work in IE11.  But it looks like you can change this to the following:

```suggestion
    word-wrap: break-word;
    overflow-wrap: break-word;
```